server/handlers.go

```diff
@@ -80,29 +81,36 @@ func validateConfig(config *explorerlib.Config) error {
 	if config.AnalyticsToken == "" {
 		return errors.New("analyticsToken Missing")
 	}
-	if config.TimeToLive == 0 {
-		return errors.New("timeToLive Missing")
+	if config.TimeToLive <= 0 {
+		scenariolib.Warning.Print("TimeToLive should be greater of equal to 1, will be set to default value of 1" )
+		config.TimeToLive = 1
 	}
-	if config.AverageNumberOfWordsPerQuery == 0 {
+	if config.AverageNumberOfWordsPerQuery < 1 || config.AverageNumberOfWordsPerQuery > 20 {
+		scenariolib.Warning.Print("AverageNumberOfWordsPerQuery out of bounds, will be set to default value of 1")
 		config.AverageNumberOfWordsPerQuery = 1
 	}
-	if config.DocumentsExplorationPercentage == 0 {
+	if config.DocumentsExplorationPercentage <= 0 || config.DocumentsExplorationPercentage > 1 {
+		scenariolib.Warning.Print("DocumentsExplorationPercentage out of bounds, will be set to default value of 0.01")
 		config.DocumentsExplorationPercentage = 0.01
 	}
-	if config.NumberOfQueryByLanguage == 0 {
+	if config.NumberOfQueryByLanguage < 0 || config.NumberOfQueryByLanguage > 200 {
+		scenariolib.Warning.Print("NumberOfQueryByLanguage out of bounds, will be set to default value of 10")
 		config.NumberOfQueryByLanguage = 10
 	}
-	if config.FetchNumberOfResults == 0 {
+	if config.FetchNumberOfResults < 0 || config.FetchNumberOfResults > 2000 {
+		scenariolib.Warning.Print("FetchNumberOfResults out of bounds, will be set to default value of 1000")
 		config.FetchNumberOfResults = 1000
 	}
 	if config.FieldsToExploreEqually == nil || len(config.FieldsToExploreEqually) == 0 {
+		scenariolib.Warning.Print("FieldsToExploreEqually out of bounds, will be set to default value of @syssource")
 		config.FieldsToExploreEqually = []string{"@syssource"}
 	}
 	if config.OutputFilePath == "" {
+		scenariolib.Warning.Print("OutputFilePath undefined, will be set to :", config.Id.String() + ".json" )
```

Comment:
Is is possible to change this to a string literal? Like `"... will be set to: %v.json", config.Id.String())` ? Ignore if it is not possible :)